geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import Canvas from "@metacell/geppetto-meta-ui/3d-canvas/Canvas";
+import CameraControls from "@metacell/geppetto-meta-ui/camera-controls/CameraControls";
+import SimpleInstance from "@metacell/geppetto-meta-core/model/SimpleInstance";
+import { withStyles } from '@material-ui/core';
+import neuron from './SketchVolumeViewer_SAAVR_SAAVR_1_1_0000.obj';
+import Button from "@material-ui/core/Button";
+import { onSelection, dataMapping } from "./SelectionUtils";
+
+const instanceTemplate = {
+  "eClass": "SimpleInstance",
+  "id": "ANeuron",
+  "name": "The first SimpleInstance to be render with Geppetto Canvas",
+  "type": { "eClass": "SimpleType" },
+  "visualValue": {
+    "eClass": GEPPETTO.Resources.OBJ,
+    'obj': neuron
+  }
+}
+
+function loadInstances (){
+  GEPPETTO.ModelFactory.cleanModel();
+  const instance = new SimpleInstance(instanceTemplate)
+  window.Instances = [instance]
+  GEPPETTO.Manager.augmentInstancesArray(window.Instances);
+}
+
+function getProxyInstances () {
+  return window.Instances.map(i => ({ instancePath: i.getId(), color: { r: Math.random(), g: Math.random(), b: Math.random(), a: 1 } }))
+}
+
+const styles = () => ({
+  container: {
+    height: '800px',
+    width: '1240px',
+    display: 'flex',
+    alignItems: 'stretch',
+  },
+});
```

Comment:
beware of formatting, there should be an empty line

Reply:
Also not picked by eslint. #63